parachain-template/runtime/src/lib.rs

```diff
@@ -225,11 +225,14 @@ const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(
 
 /// Maximum number of blocks simultaneously accepted by the Runtime, not yet included
 /// into the relay chain.
+#[cfg(feature = "parameterized-consensus-hook")]
```

Comment:
we don't actually need feature-flags in the parachain-template, as it's meant to be example code. So it should use the most recently available logic.